terraform/modules/testing/test_database.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_glue_catalog_database" "test_analytical_dataset_generation" {
+  name        = "test_analytical_dataset_generation"
+  description = "Database for the Analytical E2E Tests"
+}
```

Comment:
Does a glue database accept tags? If so, we need common_tags on it. If not, all good.

Reply:
Can confirm it doesn't take tags.

So :+1: